test/providers.test.ts

```diff
@@ -61,6 +62,18 @@ describe('loadApiProvider', () => {
     expect(provider).toBeDefined();
   });
 
+  it('should load ChatGPT-4o-latest provider', async () => {
```

Comment:
Let's skip this test. I don't think we need it.

Reply:
Fair point, I just had this in there to make sure it wasn't barfing.